### ISSUE TITLE
The current fee unit used is BTC/kB. Popular convention is Sat/B [enhancement]

### ISSUE THREAD

**Issue by cluelessperson** [spesmilo/electrum#2490]
My current max fee is 0.0043584 BTC/kB

This is tedious (math is hard) to translate to Sat/B for users, which most sites use by popular convention. This will alleviate further fee confusion and help people conform to the same known glossary.

Examples:

https://bitcoinfees.21.co/
https://blockchain.info/tx/68973cd03a6d264c1396cb1e60f27fe87214f90aa20e70abb39cc201c706a1c5

**Comment by dabura667** [comment]
you can change the issue title and edit your original post... there is no need for a new issue.

**Comment by juscamarena** [comment]
sat/byte should be standard imo.

**Comment by ecdsa** [comment]
I agree.
we could also add the color codes used in https://jochen-hoenicke.de/queue/#24h

**Comment by neocogent** [comment]
I have a commit for adding sat/byte on my repo for my own use (my-tweaks branch). I avoided submitting a PR on this because I thought the general consensus was that it was not wanted. I'd be happy to submit a PR on it so that I don't need to keep my own version. Here is link to commit - only one line changed anyway.

https://github.com/neocogent/electrum/commit/2274c312ea390169ade73fabb1764b3db8fbce7b

**Comment by au** [comment]
Not to preach to the choir, but this is rather necessary. [Mempool](https://jochen-hoenicke.de/queue/#2h) charts and [bitcoin fee](https://bitcoinfees.21.co/) calculators all use sat/byte, which are essential for trying to figure out transactions, unless you want to rely on Blockchain.info's calculator, which is the next best option at the moment.

**Comment by au** [comment]
Personally, that commit would be tipping point for me using Electrum over Blockchain, who has a very friendly fee calculator at the moment.

Would you consider cherry picking that commit into an updated repo, and making a PR for it, @neocogent ?

**Comment by neocogent** [comment]
I've rebased as a fresh branch called fee-sat-byte and created a PR for this one line change. Even if it's not merged soon you can pull this into your local repo as a branch with:

    git fetch upstream pull/2667/head:sat-fee-byte

and then test locally.

Tested on my local wallet and looked ok on my tx dialog preview.

**Comment by philrosenthal** [comment]
IMHO the fee should be set by entering "satoshi per byte", anything else results in tedious math, or overpaying.

**Comment by dabura667** [comment]
> tedious math

0.00123456 BTC/kB = 123 satoshis/byte

What I did in my brain: "read the first 5 digits after the decimal point."

So tedious.

**Comment by neocogent** [comment]
@dabura667 
IMO The math isn't hard but if you don't use it every day then remembering what factor to use leads to errors. It's not uncommon to be looking at the amount and be thinking "now was it move 5 dec places or 3  or 4". If I did a tx every day I'd no doubt keep it fresh in mind but when doing one every couple months I always have to work it out again, which isn't hard, but then try to be sure I'm not making a silly error. So that's why I made a sat/byte PR - no chance of mistakes.
